drewclem/protege
# bug/127 fix prestart npm script

## Description

For contributors using Windows, not the Windows Subsystem for Linux (WSL), they were receiving the error `'bin' is not recognized as an internal or external command, operable program or batch file.` when running `yarn start`.

The pre script for the `start` npm script has been modified to copy a static json file which contains the contributors of the project with their current commit count as of today.

It uses the `cp` command which recognized by Linux and PowerShell

When 

## To Test

1. Test from a machine running Windows
1. Ensure all dependencies for the project have been installed by running `yarn`.
1. Run `yarn start` from a PowerShell command prompt.
1. There should be no errors and the site will build and open up in a browser window.

Note: Ensure that you've asked Drew for the Firebase developer settings for your local environment. See https://github.com/drewclem/protege#getting-started .

Closes #127
Closes #153
